Soomla/Assets/Plugins/Soomla/Profile/Config/ProfileSettings.cs

```diff
@@ -141,6 +143,22 @@ private void WriteSocialIntegrationState()
 		}
 		
 		public void OnSoomlaGUI() {
+			if (EditorUserBuildSettings.activeBuildTarget == 
+#if UNITY_5
+			    BuildTarget.iOS
+#else
+			    BuildTarget.iPhone
+#endif
```

Comment:
@EugeneButusov, did not this (#if) pattern @Blinkq had issues with? Will it work in Unity4.5?